subscription-service/src/main/kotlin/com/egm/stellio/subscription/model/Subscription.kt

```diff
@@ -16,6 +23,23 @@ data class Subscription(
     val notification: NotificationParams,
     val isActive: Boolean = true
 ) {
+    companion object {
+        fun toJson(subscriptions: List<Subscription>, includeSysAttrs: Boolean): String {
```

Comment:
I find it more natural when expressed as an extension function of `List<Subscription>` (inside a companion object of Subscription, it is not really its place). 

Moreover, it allows a more consistent use whether you have a single subscription or a list of subscriptions.

Reply:
OK